static/app-strings.json

```diff
@@ -2216,9 +2216,9 @@
   "Update or Publish Replay": "Update or Publish Replay",
   "Started": "Started",
   "Starting Soon": "Starting Soon",
-  "Streaming Now": "Streaming Now",
+  "Streaming Now": "Streaming Now",  
   "Live": "Live",
-  "Upcoming Livestreams": "Upcoming Livestreams",
+  "Upcoming Livestreams": "🔴 Upcoming Livestreams",
```

Comment:
This will make the entry unused, as the `key` (whatever in `__(xx)`) is the one that is driving localization.

Options:
1. Change it that line to `"🔴 Upcoming Livestreams": "🔴 Upcoming Livestreams"`, assuming no other `__('Upcoming Livestreams')` usages is left in the code.
2. Revert this file, we'll update later.

Reply:
Fixed. Went with option 1.